api/api_sources/sources/database/models/watercraftRiskAssessment.ts

```diff
@@ -461,10 +465,25 @@ export class WatercraftRiskAssessment extends Record implements WatercraftRiskAs
 	@OneToMany(
 		type => WatercraftJourney,
 		obj => obj.watercraftAssessment,
-		{ eager: true}
+		{ eager: true }
 	)
-	@ModelProperty({type: PropertyType.array, $ref: '#/definitions/WatercraftJourney'})
+	@ModelProperty({ type: PropertyType.array, $ref: '#/definitions/WatercraftJourney' })
 	journeys: WatercraftJourney[];
+
```

Comment:
In these scenarios where the reformatting get's applied to the entire file I will throw in a comment at the code that is changing to flag for the reviewer. 

But this is pretty clear what is changing 🙂

Reply:
Good plan to move forward with!